### PR TITLE
Documentation for Google Calendar component

### PR DESCRIPTION
**Pull request by mnestor**
**Documenation for Google Calendar component:**

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#4161

### CODE REVIEW

File: source/_components/sensor.google_calendar.markdown [hunk 1]
```diff
@@ -0,0 +1,116 @@
+---
+layout: page
+title: "Google Calendar Event"
+description: "Instructions how to use Google Calendars in Home Assistant."
+date: 2015-05-08 17:15
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: home-assistant.png
```
Comment (Landrash) [inline source/_components/sensor.google_calendar.markdown]: Change this into the Google calender logo?

File: source/_components/sensor.google_calendar.markdown [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+# Example configuration.yaml entry
+google:
+  client_id: a384yb6ia83h4y68-USE-THE-VALUE-YOU-GENERATED
+  client_secret: kuhnih-USE-THE-VALUE-YOU-GENERATED
+  track_new_calendar: no
+```
+
+Configuration variables:
+
+- **client_id** (*Required*): Use the value you generated in the Prerequisies stage
```
Comment (Landrash) [inline source/_components/sensor.google_calendar.markdown]: Prerequisies should be Prerequisites.

File: source/_components/sensor.google_calendar.markdown [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+### {% linkable_title Prerequisites %}
+
+Generate a Client ID and Client Secret on [Google Developers Console](https://console.developers.google.com/start/api?id=calendar).
+
+1. To enable this sensor in your installation, add the following to your `configuration.yaml` file:
+1. When it gets to the point of asking _Which API are you using?_ just click cancel.
+1. Click on the tab 'OAuth consent screen'
+1. Set 'Product name shown to users' to anything you want. We suggest Home-Assistant.
+1. Save this page. You don't have to fill out anything else there.
+1. Click 'Create credentials' -> OAuth client ID
```
Comment (Landrash) [inline source/_components/sensor.google_calendar.markdown]: Missing dot at the end

File: source/_components/sensor.google_calendar.markdown [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+1. Set the Application type to 'Other' and give this credential set a name then click Create
```
Comment (Landrash) [inline source/_components/sensor.google_calendar.markdown]: Missing dot at the end.

File: source/_components/sensor.google_calendar.markdown [hunk 5]
```diff
@@ -0,0 +1,116 @@
...
+1. Save the client ID and secret as you will need to put these in your configuration.yaml
```
Comment (Landrash) [inline source/_components/sensor.google_calendar.markdown]: Suggest either changing this to *configuration file* or *configuration.yaml file*.

File: source/_components/sensor.google_calendar.markdown [hunk 6]
```diff
@@ -0,0 +1,116 @@
...
+  track_new_calendar: no
+```
+
+Configuration variables:
+
+- **client_id** (*Required*): Use the value you generated in the Prerequisies stage
+- **client_secret** (*Required*): Use the value you generated in the Prerequisies stage
+- **track_new_calendar** (*Optional*): Will automatically generate a binary sensor when a new calendar is detected. The system scans for new calendars on startup. By default this is set to `True`.
+
+The next steps will require you to have Home-Assistant running.
```
Comment (Landrash) [inline source/_components/sensor.google_calendar.markdown]: Dash in Home-Assistant shouldn't be there.

File: source/_components/sensor.google_calendar.markdown [hunk 7]
```diff
@@ -0,0 +1,116 @@
...
+
+After you have it running complete the Google authentication that pops up. It will give you a url and a code to enter. This will grant your Home-Assistant service access to all the Google Calendars that the account you authenticate with can read. This is a Read-Only view of these calendars.
```
Comment (Landrash) [inline source/_components/sensor.google_calendar.markdown]: Dash in Home-Assistant shouldn't be there.

**Comment by mnestor** [comment]
@Landrash Not sure what you mean by the style differing. I just compared it to ffmpeg and made a few small changes but nothing that I could see that was "quite a bit different"

**Comment by Landrash** [comment]
Fixed a few formatting and spelling misses but your changes all in all where good :)
Nice to see a new component 🍸

**Pull request merged**